packages/css-library/src/stylesheets/modules/_m-action-link.scss

```diff
@@ -14,6 +16,7 @@ a.vads-c-action-link--blue, a.vads-c-action-link--green, a.vads-c-action-link--w
   padding: 4px 0 4px 38px;
   position: relative;
   display: inline-block;
+  @include typeset('ui');
```

Comment:
Action link is being [converted to a web component now](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2500). We will need to maintain these original styles for the time being until they can be moved to the compnent.